src/dmd/frontend.h

```diff
@@ -6251,6 +6258,245 @@ enum class PREC
     primary = 16,
 };
 
+template <typename AST>
+class ParseTimeVisitor
+{
+public:
+    void visit(typename AST::Dsymbol );
```

Comment:
Seems to be missing virtual.

Reply:
Seems as if that information is missing in the generated AST....

Reply:
No `vtblIndex`?